src/database/seeds/002_endereco.js

```diff
@@ -0,0 +1,58 @@
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('endereco').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('endereco').insert([
+        {
+          id_usuario: 4,
+          cep: "01245-888",
+          logradouro: "Avenida Vergueiro",
+          numero: "8",
+          complemento: "...",
+          bairro: "Ipiranga",
+          cidade: "São Paulo",
+          estado: "São Paulo",
+          created_at: null, 
+          updated_at: null 
+        },
+        {
+          id_usuario: 1,
+          cep: "74895-167",
+          logradouro: "Avenida Vergueiro",
+          numero: "67",
+          complemento: "Viela 2",
+          bairro: "Sacoma",
+          cidade: "São Paulo",
+          estado: "São Paulo",
+          created_at: null, 
+          updated_at: null 
```

Comment:
Deletar estas linhas caso você concorde em retirar esses atributos da tabela Endereço.
```suggestion
          estado: "São Paulo"
```